CHANGELOG.md

```diff
@@ -25,6 +25,38 @@ You can expect similar releases like this one in the coming months.
 - [Popover] Deprecate transition onX props (#22202) @mbrookes
 - [Snackbar] Deprecate transition onX props (#22206) @mbrookes
 
+## 4.11.2
+###### _Dec 2 2020_
+
+This release widens the peer dependency scope of React to accept ^17.0.0.
+The change allows developers to upgrade React independently from Material-UI. The best support for React 17 can be found on Material-UI v5.
```

Comment:
That is not true. They can already upgrade.

Reply:
Yeah, you are right, it's possible. I have updated the wording to be more accurate for #23306